### ISSUE TITLE
Internal path references don't work 

### ISSUE THREAD

**Issue by gregie156** [LucyBot-Inc/api-spec-converter#199]
The OpenAPI 3 below is valid. You can test it on https://editor.swagger.io/
However, when I try to convert it to Swagger 2, I get a stacktrace (see below).
I am on windows 10; api-spec-converter v 2.7.23

Example Specs:
```
openapi: 3.0.1
info:
  title: Sample API
  description: API description in Markdown.
  version: 1.0.0
servers:
  - url: 'https://api.example.com'
paths:
  /users:
    get:
      summary: Returns a list of users.
      description: Optional extended description in Markdown.
      responses:
        '200':
          description: OK
  /other_users:
    $ref: '#/paths/~1users'
```

Stacktrace:
```
Fatal TypeError: Cannot read property 'parameters' of undefined
    at module.exports.Converter.convertParameters (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\lib\converters\openapi3_to_swagger2.js:175:13)
    at module.exports.Converter.convertOperations (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\lib\converters\openapi3_to_swagger2.js:113:14)
    at module.exports.Converter.convert (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\lib\converters\openapi3_to_swagger2.js:32:8)
    at converters.swagger_2.Promise.method.oa (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\lib\formats\openapi_3.js:19:24)
    at tryCatcher (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\node_modules\bluebird\js\release\util.js:16:23)
    at C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\node_modules\bluebird\js\release\method.js:15:34
    at Promise.try (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\lib\base_format.js:165:12)
    at tryCatcher (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\node_modules\bluebird\js\release\util.js:16:23)
    at Function.Promise.attempt.Promise.try (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\node_modules\bluebird\js\release\method.js:39:29)
    at module.exports.BaseFormat.convertTo (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\lib\base_format.js:157:21)
    at Converter.getSpec.then.fromSpec (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\index.js:33:32)
    at tryCatcher (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\node_modules\bluebird\js\release\promise.js:512:31)
    at Promise._settlePromise (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (C:\Users\gregoryb\AppData\Roaming\npm\node_modules\api-spec-converter\node_modules\bluebird\js\release\promise.js:694:18)
```

**Comment by bobby-brennan** [comment]
Thanks for the report, as well as the minimal repro case. This was an issue with resolving refs including a `/` character (which is [apparently escaped as ~1](https://tools.ietf.org/html/rfc6901#page-3)).

Fixed now in `2.7.25`

**Comment by gregie156** [comment]
@bobby-brennan thanks for fixing this.
It's not intuitive (to say the least) that slashes are escaped as ~1. Who knows how they came up with that :)
